<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettinngWindow</class>
 <widget class="QMainWindow" name="SettinngWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>480</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>700</width>
        <height>480</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>220</y>
          <width>488</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>模型 URL   :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="self_url_lineEidt">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>270</y>
          <width>490</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>API key      :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="self_key_lineEidt">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>380</y>
          <width>471</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>自定义免费模型教程: </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>651</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>由于 token 限制，桌宠只有100次会话次数，超过100之后需要到大模型官网申请用户自己的模型</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>420</y>
          <width>471</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>桌宠开源网址: </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>661</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>硅基流动 免费模型网址: https://cloud.siliconflow.cn/me/models</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>661</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>DeepSeek 模型网址: https://www.deepseek.com/zh</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>101</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>当前剩余次数: </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>60</y>
          <width>69</width>
          <height>19</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
    color: red;
}</string>
        </property>
        <property name="text">
         <string>666</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>340</y>
          <width>271</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>温度越高,模型思考越久，回复更准确</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>180</y>
          <width>61</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>编辑</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>260</y>
          <width>61</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>488</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>模型名称    :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="self_model_lineEidt"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>330</y>
          <width>271</width>
          <height>34</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>温度   :  </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="slider">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBothSides</enum>
           </property>
           <property name="tickInterval">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>705</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_model"/>
  </widget>
  <action name="action_model">
   <property name="text">
    <string>模型设置</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
